### ISSUE TITLE
Some Anime Seasons can not be seen by the Downloader. But if it can see the Seasson, it will only display 1 Episode wich is broken.

### ISSUE THREAD

**Issue by LeTeckGas** [hama3254/Crunchyroll-Downloader-v3.0#890]
**Describe the bug**
I am trying since November 2023 to download the Anime Highshool DXD Seasson 4 to download in German. (Cause i am German and i dont understand Japanese). When i try to downlaod this Season it only shows me one single Eppisode wich i cant even download. 

**Where does the issue happen?**
 - Platform: WIndows 10 64x Crunchyroll
 - Settings [see Screenshots]
 - Version: 1.17.1

**Screenshots**
![Deutsch](https://github.com/hama3254/Crunchyroll-Downloader-v3.0/assets/83094268/0d8f9323-339f-42bd-bc3b-b46a596fad04)
![Japanese](https://github.com/hama3254/Crunchyroll-Downloader-v3.0/assets/83094268/21988571-bbef-42a4-9b1a-43ede32187f7)
![Main](https://github.com/hama3254/Crunchyroll-Downloader-v3.0/assets/83094268/0fa3e77a-5abb-48d5-87c9-7df27db0aea6)
![Output](https://github.com/hama3254/Crunchyroll-Downloader-v3.0/assets/83094268/5c673d59-5914-4c70-a2d0-378c72fca259)
![Naming ong](https://github.com/hama3254/Crunchyroll-Downloader-v3.0/assets/83094268/38ccc4b9-18aa-49c6-a5b5-98cba244c101)
![Crunchyroll](https://github.com/hama3254/Crunchyroll-Downloader-v3.0/assets/83094268/cb5b9733-2a3a-4dde-bf0a-69f9340a702e)
![Funimation](https://github.com/hama3254/Crunchyroll-Downloader-v3.0/assets/83094268/79f9c254-aefd-454c-ba02-f6f370fc443f)
![About](https://github.com/hama3254/Crunchyroll-Downloader-v3.0/assets/83094268/e3d89831-35df-457d-8e0e-82ce8b1cdc03)

**Additional context**
Deutsch means German.

**Comment by hama3254** [comment]
Looking into this issue i found that the downloader filters all German audio episodes.
The filter was placed into the Downloader because i use the API for there website where they want to display all episodes even tho not all have the right dub. The downloader should to it differnt, if you select `Deutsch` than you don't expect that it switches in the middle (as CR would display it on there website).

I wrote that in the past to check the first episode for it's dub an than only display the same dub moving forward.
This time `Episode 0` is japanese only while 1 to 12 has ger Dub but japanese has been uses a dub for that episode list now so german episodes where not added.

i need to see how i filter the website output correctly with such cases in mind.

**Comment by LeTeckGas** [comment]
Thanks for the Reply Hama, but I don't quite understand what you mean. Do I do something wrong or is this some sort of a Bug?

**Comment by hama3254** [comment]
> is this some sort of a Bug?

Yes this is a bug, the downloader checks the default dub of episodes and in this case the first dub is japanese so the german dub that we want gets discarded.

You can still download the show by either downloading each episode by hand or you select `Japanese` and enable the `override Dub language`  (set to german ofcourse)

![grafik](https://github.com/hama3254/Crunchyroll-Downloader-v3.0/assets/52826614/b7f67323-1711-49db-b445-65e27d54d0b5)

**Comment by LeTeckGas** [comment]
Ok, but how can I Download Episodes by Hand?

**Comment by hama3254** [comment]
This issue is only with the season processing, you should be able to open each episode in your web browser and copy that url into the add window instead of the URL from the overview page.

**Comment by hama3254** [comment]
you can try the new [v3.17.2 builld](https://github.com/hama3254/Crunchyroll-Downloader-v3.0/releases/tag/v3.17) which should resolve this issue.

**Comment by LeTeckGas** [comment]
Thank you Hama. God bless you.
Have a nice day.
